CONTRIBUTING.md

```diff
@@ -151,6 +151,10 @@ functionality (e.g. a performance optimization) and new tests are not required.
 In general, including tests with your pull request dramatically increases the
 chances it will be accepted.
 
+### Leveraging Docker for Testing
```

Comment:
I'm wondering if the title for this section might be too limiting for its purpose, since Docker could be used to aid development environments too. What are your thoughts about changing the framing/scope?

Reply:
I love it! I took a pass with 7132fc1.